### DISCUSSION TITLE
`on_right_click` property for `Container`

### DISCUSSION THREAD

**Discussion by taaaf11** [flet-dev/flet#2753] [Q&A] [answered]
**Feature Request**

`on_right_click` property for `Container`. Consequently, a right click menu (may be a popup menu?) should be displayed

**Comment by ndonkoHenri** [comment, marked as answer]
Use the GestureDetector for that: https://flet.dev/docs/controls/gesturedetector
Event handlers with "secondary" are for the right clicks.
